src/mqt/ionshuttler/multi_shuttler/gate_partitioning_tabu.py

```
@@ -0,0 +1,1535 @@
+    ) + (
+        (after_dst_count - capacity) * (load_vector[dst_cluster] + supernode_load)
+        if after_dst_count > capacity
+        else 0.0
+    )
+    return float(after - before)
+
+
+def _distance_delta(
+    slice_index: int,
+    qubits: Sequence[int],
+    current_cluster: int,
```

Ruff (0.15.12)

[warning] Unused function argument: `current_cluster`

(ARG001)